package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "subscribe-ui-event",
-  "version": "0.2.2",
+  "version": "0.2.3",
```

Comment:
typically we don't bump version with PR, 

we will do

```
npm version patch   (will generate tags)
git push origin master 
git push origin --tags 
npm publish ./
```

so that we can get the tag